src/Elastic.Documentation.Configuration/ConfigurationFileProvider.cs

```diff
@@ -4,14 +4,26 @@
 
 using System.IO.Abstractions;
 using Microsoft.Extensions.DependencyInjection;
+using NetEscapades.EnumGenerators;
 
 namespace Elastic.Documentation.Configuration;
 
+[EnumExtensions]
+public enum ConfigurationSource
+{
+	Local,
```

Comment:
Can we use local and put the config in elastic/docs-internal-workflows?

Reply:
We could, as source of truth or as a copy?

I think we want the source of truth to remain in docs-builder.

Reply:
I think config should live where its read. 

If it lives in elastic/docs-internal-workflows we'd need multiple copies for each environment. 

We can do a lot more PR validations from this repository too IMO.  Let's discuss today :)